versioned_docs/version-0.8/Guides/backups.md

```diff
@@ -70,3 +70,14 @@ Run `crontab -e` and add the following line:
 
 This will create a backup at /var/www/controlpanel/backups-$(date +\%F).sql every day at midnight.
 Every file will have the date when it was made in the filename, so you can also resolve issues that happened a few days ago.
+
```

Comment:
Please follow the structure of the file like this:

# Application Key
<Smal explenation what it is and why its important to keep save>
### Appliaction key backup
<Text>
###Apllication key restore
<Text>